### DISCUSSION TITLE
Tips and Tricks

### DISCUSSION THREAD

**Discussion by aounalazzam** [pocketbase/pocketbase#4788] [Q&A]
Is There A Tips and Tricks to make pocketbase more faster and performant and more secure? 

i know this is might be wird 🙄 question but to know how to make scale as can

**Comment by byayex** [comment]
I mean if you need to, you can do the following things:
- Use Indexes
- Get better storage (nvme for example)

In my opinion these are the most important things.

If you need more informations about this feel free to look it up here
-> [https://developer.android.com/topic/performance/sqlite-performance-best-practices?hl=de](https://developer.android.com/topic/performance/sqlite-performance-best-practices?hl=de)

Now coming to the secure part of your question.

You just have to take care about the api rules you can set in the UI. 
-> [https://pocketbase.io/docs/api-rules-and-filters/](https://pocketbase.io/docs/api-rules-and-filters/)
